ambry-clustermap/src/test/java/com.github.ambry.clustermap/HelixClusterManagerTest.java

```diff
@@ -514,6 +514,13 @@ public void xidTest() throws Exception {
       assertFalse(instanceName.equals(aheadInstanceConfig.getInstanceName()));
     }
 
+    // Ahead instance should be honored if the cluster manager is of the aheadInstance.
+    HelixClusterManager aheadInstanceClusterManager =
+        new HelixClusterManager(clusterMapConfig, aheadInstanceConfig.getInstanceName(), helixManagerFactory,
+            new MetricRegistry());
+    assertEquals(instanceCount, aheadInstanceClusterManager.getDataNodeIds().size());
+    aheadInstanceClusterManager.close();
+
```

Comment:
Make sure the `clusterManager` is closed at the end of this test.

Reply:
Ah, good catch. The intention was to use the member variable named `clusterManager` (which gets closed automatically in @After). Made changes to do that.